srcpkgs/vala/template

```diff
@@ -28,6 +28,13 @@ pre_configure() {
 	autoreconf -fi
 }
 
+post_install() {
+	vsed -e 's|^datadir=.*|datadir=${pc_sysrootdir}/${datarootdir}|g' \
+		 -e 's|^vapidir=.*|vapidir=${pc_sysrootdir}/${datadir}/vala/vapi|g' \
+		 -e 's|^vapidir_versioned=.*|vapidir_versioned=${pc_sysrootdir}/${datadir}/vala-0.42/vapi|g' \
+		 -i ${DESTDIR}/usr/lib/pkgconfig/vapigen-0.42.pc
+}
```

Comment:
Although it is obvious what this does if one knows pkg-config a comment wouldn't hurt